queries/query_builders.go

```diff
@@ -180,35 +180,41 @@ func buildUpdateQuery(q *Query) (*bytes.Buffer, []interface{}) {
 	return buf, args
 }
 
+func writeParameterizedModifiers(q *Query, buf *bytes.Buffer, args *[]interface{}, m modifiers) {
+	argsLen := len(*args)
+	modBuf := strmangle.GetBuffer()
+	fmt.Fprintf(modBuf, m.keyword())
+
+	for i, j := range m.clauses() {
+		if i > 0 {
+			fmt.Fprintf(modBuf, m.delim())
+		}
+		fmt.Fprintf(modBuf, j.clause)
+		*args = append(*args, j.args...)
+	}
+
+	var resp string
+	if q.dialect.UseIndexPlaceholders {
+		resp, _ = convertQuestionMarks(modBuf.String(), argsLen+1)
+	} else {
+		resp = modBuf.String()
+	}
+
+	fmt.Fprintf(buf, resp)
```

Comment:
Although I realize the original code had `fmt.Fprintf` I'm wondering why now. Should these not be `io.WriteString`?

Reply:
I'm still learning Go, so I don't yet know what is idiomatic.  But `io.WriteString` seems to work here.  Note that there's a lot of `fmt.Fprintf` statements elsewhere in the file.  Should these be picked up as well?